templates/cisco_wlc_ssh_show_mobility_sum.textfsm

```diff
@@ -0,0 +1,23 @@
+Value MAC_ADDR (([\d1-9a-f]{2}\:?){6})
+Value IP_ADDR (([\d1-9]+\.?){4})
+Value GROUP_NAME (\S+)
+Value MULTICAST_IP (([\d1-9]+\.?){4})
+Value STATUS (.+?)
+
+Start
+  ^Mobility\s+Protocol\s+Port
+  ^Default\s+Mobility\s+Domain
+  ^Multicast\s+Mode\s+
+  ^Mobility\s+Domain\s+ID\s+for\s+802.11r
+  ^Mobility\s+Keepalive\s+Interval
+  ^Mobility\s+Keepalive\s+Count
+  ^Mobility\s+Group\s+Members\s+Configured
+  ^Mobility\s+Control\s+Message\s+DSCP\s+Value
+  ^Controllers\s+configured\s+in\s+the\s+Mobility\s+Group -> Mobility_Controllers
```

Comment:
I would make the state change to the actual headers we're looking to obtain data like you did under `Mobility_Controllers`, but move it up here to transition states.

Reply:
Ok I think I corrected what you asked.